__tests__/app.test.js

```diff
@@ -89,3 +90,35 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200: should return an articles array of article objects with their properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toBeInstanceOf(Array);
+        response.body.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+          expect(article).not.toHaveProperty("body");
+          expect(typeof article.votes).toBe("number");
+        });
```

Comment:
Whenever we use a loop to assert items exist in an array we need to make sure we're not getting a false positive with an empty array. So please also assert that the arr isn't empty.